13/umbraco-deploy/getting-started/deploy-settings.md

```diff
@@ -448,7 +458,85 @@ If set to `true` the configuration details shown on the setting's dashboard will
 
 If set to `true` the version details shown on the setting's dashboard will be hidden.
 
-## Configuration Via Code
+### ValidateDependenciesOnImport
+
+A default notification handler for the `ValidateArtifactImportNotification` is registered by Deploy that:
+
+* Adds warnings for dependencies that must match exactly, and are both not in the import and not with matching checksums in the target environment
+* Adds warnings for all remaining content dependencies and errors for all schema dependencies that don't exist in the import
+
+To avoid this handler from being registered, you can set this setting to `false`.
+
+### Import on startup
```

Comment:
This is all the information that we want, but I wonder if it might be better to split it so we have some information on the "Import/Export" page itself, and we keep this to just the detail about configuration.

At very least we should have some reference to this feature on the "Import/Export" page, so make it discoverable from there.

So perhaps for this page we have something like:

```
Deploy can [import content and/or schema previously exported from another Umbraco installation on start-up](../deployment-workflow/import-export.md#<anchor link to new section>).

If you want to customize the default behaviour you can use these settings:

<bullet points for settings>
```

And then in a new section on the import/export page add everything else you have here (why you might want to use it, the description for the default behaviour and the code sample for extension).  Apart from the details of the settings where you replace with something like.

```
If you want to [customize the default behaviour you do so via settings](../getting-started/deploy-settings.md#<anchor link to settings>).
```

Maybe even better rather than it being a section on the import/export page, it's a new sub-page, like we currently have for "Import with migrations".

Reply:
Having it on a dedicated page might indeed make the feature more discoverable indeed, I'll create the page, move things and create links between them 👍🏻